src/arch/runtime/context_switching.cc

```diff
@@ -292,7 +289,7 @@ artificial_stack_t::artificial_stack_t(void (*initial_fun)(void), size_t _stack_
 #elif defined(__s390x__)
     sp -= 16; // r6-r13 and f8-f15.
 #elif defined (__powerpc64__)
-    sp += 24;
+    sp -= 20; // r14-r31, toc, cr.
```

Comment:
There's a comment above that indicates that a context switch "doesn't swap the floating-point registers"